src/main/lrsql/ops/command/admin.clj

```diff
@@ -44,3 +45,31 @@
       (bp/-delete-admin-account! bk tx input)
       {:result (:account-id input)})
     {:result :lrsql.admin/missing-account-error}))
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Ensure Admin Account from OIDC
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(s/fdef ensure-admin-oidc!
+  :args (s/cat :bk ads/admin-backend?
+               :tx transaction?
+               :input ads/ensure-admin-oidc-input-spec)
+  :ret ads/ensure-admin-ret-spec)
+
+(defn ensure-admin-oidc!
+  "Create a new admin with OIDC issuer or verify issuer of an existing admin.
+  Returns a map where `:result` is the account ID."
+  [bk tx {:keys [username oidc-issuer]
+          :as   ensure-input}]
+  (if-let [{extant-issuer :oidc_issuer ;; TODO: is not coerced to kebab?
```

Comment:
It is indeed _not_ coerced to kebab! Notice in other query functions that you see the same thing (e.g. https://github.com/yetanalytics/lrsql/blob/4fa3cc460aa1e896bd15d5efa9b1ffe3f2ced4d7/src/main/lrsql/ops/query/document.clj#L30-35)

Reply:
Cool, I'll remove the comment

Reply:
https://github.com/yetanalytics/lrsql/pull/206/commits/a13f5f549c73962ae5214ee69e1e526dd6679a0a